### ISSUE TITLE
Disable/Skip HTML single-line comment 

### ISSUE THREAD

**Issue by ghost** [beautify-web/js-beautify#426]
I usually have a comment after the closing html tag in order to identify the tag on big files with a lot of tags.

Input:

```
<div class='panel'>
    ...
</div><!-- /.panel -->
```

Current output:

```
<div class='panel'>
    ...
</div>
<!-- /.panel -->
```

Desired output:

```
<div class='panel'>
    ...
</div><!-- /.panel -->
```

It would be nice if it would be able skip those single-line comments.

**Comment by cjaoude** [comment]
+1.

Sooner the better! :) much appreciated.

**Comment by olsonpm** [comment]
+1.  My pragmatic use-case is the freaking html inline block spacing behavior.  Basically you in order to remove auto white-space between inline elements you need their tags to be together, which I normally accomplished via comments.  The other workaround which has been driving me nuts was to set font size to zero. Now instead of inheriting a reasonable font-size, I inherit the font-size of zero and often forget about it days later causing me to say "hey, why isn't that element showing up?"

**Comment by olsonpm** [comment]
For those interested, I couldn't handle the "font-size: 0" hack anymore so I customized my beautify-html to the following:

```
// beautify-html.js, version 1.5.4, line 662

// CUSTOM
//
// Don't add newlines before or after comments
//
var endComment = (this.token_text.slice(0, 4) === '<!--');
var lastTokenText = this.output[this.output.length - 1];
var beginningComment = (lastTokenText.slice(-3) === '-->');
if (!endComment && !beginningComment) {
    arr.push('\n');
}
```
The simple test I created was:
```
var commentIn = '<div class="ok"></div><!--\ndo not format\n--><div class="ok"></div>';
var commentExpected = '<div class="ok"></div><!--\ndo not format\n--><div class="ok"></div>';
assert.strictEqual(html_beautify(commentIn, opts), commentExpected);
```

The above is not tested very well, and I don't have time to spend making it suitable for an official PR.  I will update the code here if I find any bugs.

**Comment by stevgouws** [comment]
Hi any news on if this option will be added? Would be great. Thanks.

**Comment by THavas** [comment]
+1 yes please. Love Beautify but hate having my ending div comments on a new line.

**Comment by mazadah** [comment]
Was this resolved? I am interested in this.

**Comment by bitwiseman** [comment]
@mazadah 
This is still open and waiting to be implemented.  The three-dash structure proposed by @olsonpm would be one way to do this to start with.  That change would be non-breaking, but would require people to do something different with their html. 

Having single-line comments (comments whose contents do not include a newline) stay on the same line would also be reasonable.  It might also be reasonable to have the rule be that a closing tag followed by a comment, followed by a newline should not be moved.

**Comment by bitwiseman** [comment]
This is fixed in 1.8.0 as #1337
